docs/src/tutorials/getting_started/tolerances.jl

```diff
@@ -360,3 +366,109 @@ is_solved_and_feasible(model)
 # algorithm. There will always be models in the gray boundary at the edge of
 # feasibility, for which the question of feasibility is not a clear true or
 # false.
+
+# ## Problem scaling
+
+# Problem scaling refers to the absolute magnitudes of the data in your problem.
+# We say that a problem is poorly scaled if there are very small ($< 10^{-3}$)
+# or very large ($> 10^6$) coefficients in the problem, or if the ratio of the
+# largest to smallest coefficient is large.
+
+# Numerical issues related to the feasibility tolerances most commonly arise
+# because of poor problem scaling.
+
```

Comment:
I'd say here that the next examples assume a tolerance of 1e-8, but actual tolerances may vary from on solver to another